go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.0
-	github.com/containers/buildah v1.19.4
-	github.com/containers/common v0.34.3-0.20210208115708-8668c76dd577
+	github.com/containers/buildah v1.19.0
+	github.com/containers/common v0.33.1-0.20210130125707-da196fb0f743
```

Comment:
Do we really want to go from c/common 34 to 33 on master?

Reply:
No I cut a new release today on common. we should move to that.

Reply:
@TomSweeneyRedHat  Oh this is just to force CI to use journald logging on the tests, the vendor commit will be dropped once the tests pass. The common here defaults to journald logging, which is what we want tests to pass on.